baybe/constraints/base.py

```diff
@@ -93,6 +97,12 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:
             The dataframe indices of rows where the constraint is violated.
         """
 
+    def summary(self) -> dict:  # noqa: D102
```

Comment:
if you add `"""See base class."""` instead of `# See base class.` and remove the exception the linter will not complain

Reply:
I did it this way to make it consistant with the other abstract functions (eg. the `is_in_range` one)
![image](https://github.com/emdgroup/baybe/assets/118910837/a393b16e-7d5f-475d-a023-b8f77a841305)
@AVHopp what do you think about this exception?

Reply:
i think the doc strings there are wrong, they should not sue an exception and use `""" """` comments aka docstrings

Reply:
We need to have it with the # sign. This will results in the documentation pulling the docstring from the parent class. Using triple backticks would replace the docstring by one with the content of what is then written within the backticks

Reply:
I see, forgot about that, ignore my comment then